https://bugzilla.suse.com/show_bug.cgi?id=1216830
https://github.com/rspamd/rspamd/pull/4732

The caller uses the value to size its memory allocation:
        gsize niov = rspamd_log_fill_iov(NULL, now, module, id, function, level_flags, message,
                                                                         mlen, rspamd_log);
        struct iovec *iov = g_alloca(sizeof(struct iovec) * niov);
and that faulted on aarch64.

Index: rspamd-3.7.5/src/libserver/logger/logger.c
===================================================================
--- rspamd-3.7.5.orig/src/libserver/logger/logger.c
+++ rspamd-3.7.5/src/libserver/logger/logger.c
@@ -1046,25 +1046,7 @@ gsize rspamd_log_fill_iov(struct iovec *
 	gint r;
 
 	if (iov == NULL) {
-		if (log_rspamadm) {
-			if (logger->log_level == G_LOG_LEVEL_DEBUG) {
-				return 4;
-			}
-			else {
-				return 2; /* No time component */
-			}
-		}
-		else if (log_systemd) {
-			return 4;
-		}
-		else {
-			if (log_color) {
-				return 5;
-			}
-			else {
-				return 4;
-			}
-		}
+		return 5;
 	}
 	else {
 		static gchar timebuf[64], modulebuf[64];
